### PR TITLE
feat: update delivery for research filtered variants

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Removed:
- Rankscore from storing

Added:
- store all research level VCFs

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Review by khurrammaqbool — APPROVED** [review]
👍 . 
It may be nice to have some output from test runs.

File: BALSAMIC/constants/workflow_rules.py [hunk 1]
```diff
@@ -117,11 +117,18 @@
...
     "vep_stat",
+    "bcftools_filter_TNscope_umi_tumor_only",
+    "bcftools_filter_TNscope_umi_tumor_normal",
     "bcftools_filter_vardict_tumor_only",
     "bcftools_filter_vardict_tumor_normal",
     "bcftools_filter_tnscope_tumor_only",
     "bcftools_filter_tnscope_tumor_normal",
+    "bcftools_filter_tnhaplotyper_tumor_only",
+    "bcftools_filter_tnhaplotyper_tumor_normal",
     "bcftools_filter_manta",
```
Comment (ashwini06) [inline BALSAMIC/constants/workflow_rules.py:128]: As the rules names are same https://github.com/Clinical-Genomics/BALSAMIC/blob/a3ce64a7861518f5bb76b16477d6797e76470f53/BALSAMIC/snakemake_rules/annotation/varcaller_sv_filter.rule#L8

https://github.com/Clinical-Genomics/BALSAMIC/blob/a3ce64a7861518f5bb76b16477d6797e76470f53/BALSAMIC/snakemake_rules/annotation/varcaller_sv_wgs_filter_tumor_normal.rule#L8

will there be any conflict?

File: BALSAMIC/constants/workflow_rules.py [hunk 2]
```diff
@@ -117,11 +117,18 @@
...
+    "bcftools_filter_delly",
```
Comment (ashwini06) [inline BALSAMIC/constants/workflow_rules.py:129]: same for delly?

File: BALSAMIC/constants/workflow_rules.py (hunk 1 above)
Comment (hassanfa) [reply BALSAMIC/constants/workflow_rules.py:128]: No this won't be a conflict. Rules are included in https://github.com/Clinical-Genomics/BALSAMIC/blob/develop/BALSAMIC/constants/workflow_rules.py

I think it'll be best to rename `varcaller_sv_filter.rule` `varcaller_sv_panel_wes_filter.rule` at some point to clearly reflect what type of task it is trying to achieve.

Comment (ashwini06) [reply BALSAMIC/constants/workflow_rules.py:128]: Sure, let's merge this for now. And I will refactor on this renaming part later on

**Pull request merged**